### PR TITLE
PLAT-114 migrate get trip events to ingest service

### PR DESCRIPTION
**Pull request by nessur**
## 📚 Purpose
*[description/approach]*

## 👌 Resolves:
- [ ] ✨ PLAT-114 implements new feature

## 📦 Impacts:
*[mds-ingest-service]*

## ✅ PR Checklist
- [x] write tests

### CODE REVIEW

File: packages/mds-ingest-service/repository/index.ts [hunk 1]
```diff
@@ -355,6 +356,48 @@ class IngestReadWriteRepository extends ReadWriteRepository {
...
+      const connection = await this.connect('rw')
+      const tripIdQuery = await connection
+        .getRepository(EventEntity)
+        .createQueryBuilder()
+        .distinctOn(['trip_id'])
+        .limit(take)
+      if (start_time) tripIdQuery.where('start_time >= :start_time', { start_time })
+      if (end_time) tripIdQuery.andWhere('end_time <= :end_time', { end_time })
+      if (provider_id) tripIdQuery.andWhere('provider_id = :provider_id', { provider_id })
+      if (skip) tripIdQuery.andWhere('trip_id > :trip_id', { skip })
```
Comment (avatarneil) [inline packages/mds-ingest-service/repository/index.ts:371]: If `start_time` is undefined, will these `andWhere` clauses work properly?

Comment (nessur) [reply packages/mds-ingest-service/repository/index.ts]: they do! thankfully. I tried a few more configurations, and this is the least awkward.

File: packages/mds-ingest-service/@types/index.ts [hunk 2]
```diff
@@ -109,6 +109,14 @@ export type GetVehicleEventsResponse = {
   }
 }
 
+export interface ReadTripEventsQueryParams {
+  skip?: UUID
```
Comment (drtyh2o) [inline packages/mds-ingest-service/@types/index.ts:113]: Decided on skip/take over limit/cursor here?

**Review by drtyh2o — APPROVED** [review]

**Review by avatarneil — APPROVED** [review]
LGTM!

File: packages/mds-ingest-service/@types/index.ts (hunk 2 above)
Comment (nessur) [reply packages/mds-ingest-service/@types/index.ts:113]: Yeah, this was a direct translation of the original mds-db query we wrote months ago, so I didn't have any reasons to refactor the interface or behavior in isolation of changes to the trip-log report processor.

**Pull request merged**